File: TriviaServer/TriviaServer/LeaveGameResponse.cpp
#include "LeaveGameResponse.h"

LeaveGameResponse::LeaveGameResponse(int status)
{
	this->messageCode = 87;
	this->_status = status;
}

nlohmann::json LeaveGameResponse::castToJson() const
{
	return nlohmann::json{ {"status" , _status} };
}

File: TriviaWPFClient/TriviaClient/LeaveGameResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TriviaClient
{
    class LeaveGameResponse : Response
    {
        public const int MESSAGE_CODE = 87;
        LeaveGameResponse() : base(MESSAGE_CODE){}
        public int status { get; set; }
    }
}
